rcode-reader/src/main/java/eu/rssw/pct/elements/DataSourceElement.java

```diff
@@ -0,0 +1,62 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Set;
+
+import eu.rssw.pct.AccessType;
+import eu.rssw.pct.RCodeInfo;
+
+public class DataSourceElement extends AbstractAccessibleElement {
+  // private int flags;
+  // private int crc;
+
+  private String queryName;
+  private String keyComponentNames;
+  private String[] bufferNames;
+
+  public DataSourceElement(String name, Set<AccessType> accessType, byte[] segment, int currentPos, int textAreaOffset,
+      ByteOrder order) {
+    super(name, accessType);
+
+    int bufferCount = ByteBuffer.wrap(segment, currentPos, Short.BYTES).order(order).getShort();
+    // this.flags = ByteBuffer.wrap(segment, currentPos + 2, Short.BYTES).order(order).getShort();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)